src/shared/components/CropPopup/CropPopup.tsx

```diff
@@ -10,30 +11,45 @@ import { getUploadFormData } from 'shared/utils';
 
 import { StyledCropWrapper } from './CropPopup.styles';
 import { CropPopupProps } from './CropPopup.types';
-import { cropOptions } from './CropPopup.const';
+import { cropImage, initPercentCrop } from './CropPopup.utils';
 
-export const CropPopup = ({ open, setCropPopupVisible, setValue, image }: CropPopupProps) => {
+export const CropPopup = ({
+  open,
+  setCropPopupVisible,
+  setValue,
+  image,
+  ratio = 1,
+}: CropPopupProps) => {
   const { t } = useTranslation('app');
-  const cropperRef = useRef<ReactCropperElement>(null);
+
+  const [crop, setCrop] = useState<Crop>();
+  const imgSrc = useMemo(() => URL.createObjectURL(image), [image]);
```

Comment:
Do y need memo here?

Reply:
Yes, without memo there are a lot of re-renders